napari/utils/events/event.py

```diff
@@ -511,7 +511,21 @@ def _normalize_cb(self, callback) -> Union[CallbackRef, Callback]:
         # make the connection without making a strong reference to the
         # instance.
         if inspect.ismethod(callback):
+            old_callback = callback
             callback = (callback.__self__, callback.__name__)
+            if (
+                not hasattr(callback[0], callback[1])
+                or getattr(callback[0], callback[1]) != old_callback
+            ):
+                for name in dir(callback[0]):
+                    meth = getattr(callback[0], name)
+                    if inspect.ismethod(meth) and meth == old_callback:
+                        callback = callback[0], name
+                        break
+                else:
+                    raise RuntimeError(
+                        f"During bind method {callback[1]} of object {callback[0]} an error happen"
```

Comment:
```suggestion
                        f"The method '{callback[1]}' of object '{callback[0]}' could not be found"
                        f" while binding '{old_callback}'"
```

Reply:
by the way, @Czaki, can you think of a case where this might happen? I notice that the test does not cover this line, so I wonder if the entire try/except can be removed.

Reply:
test added